manager/integration/tests/test_ha.py

```diff
@@ -755,16 +751,10 @@ def test_rebuild_with_restoration(
 
     # cleanup
     backupstore_cleanup(client)
-    delete_and_wait_pod(core_api, original_pod_name)
-    delete_and_wait_pvc(core_api, original_pvc_name)
-    delete_and_wait_pv(core_api, original_pv_name)
-    delete_and_wait_pod(core_api, restore_pod_name)
-    delete_and_wait_pvc(core_api, restore_pvc_name)
```

Comment:
The PV and PVC of the restore volume are created by `create_pv_for_volume` and `create_pvc_for_volume`. Need to check if this cleanup can be removed.

Reply:
In this case, the pod is getting created with pod_make fixture and as we have PV/PVC cleanup there. These delete methods are not required.